reporting-cli/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "reporting-cli",
+  "description": "Download reports from anywhere",
+  "main": "index.js",
+  "type": "module",
+  "dependencies": {
+    "aws-sdk": "^2.1237.0",
+    "commander": "^9.4.0",
+    "dompurify": "^2.1.1",
+    "dotenv": "^16.0.3",
+    "jsdom": "^19.0.0",
+    "lodash": "^4.17.21",
+    "nodemailer": "^6.8.0",
+    "nodemailer-express-handlebars": "^5.0.0",
+    "ora": "^6.1.2",
+    "puppeteer": "^19.1.1"
```

Comment:
since it's puppeteer, should limitations also mention the environments supported? e.g. i don't think it supports linux arm64